internal/pkg/cli/app_init.go

```diff
@@ -229,7 +233,24 @@ func (o *initAppOpts) createManifest() (string, error) {
 	return manifestPath, nil
 }
 
-func (o *initAppOpts) createLoadBalancedAppManifest() (*manifest.LoadBalancedWebApp, error) {
+func (o *initAppOpts) newManifest() (encoding.BinaryMarshaler, error) {
+	switch o.AppType {
+	case manifest.LoadBalancedWebApplication:
+		return o.newLoadBalancedWebAppManifest()
+	case manifest.BackendApplication:
+		return manifest.NewBackendApp(manifest.BackendAppProps{
```

Comment:
Would it be better to have `newBackAppManifest()` to be consistent with `newLoadBalancedWebAppManifest()`?

Reply:
like have `o.newBackendAppManifest` that calls the constructor?

Reply:
Yeah just a nit though